src/module.ts

```diff
@@ -1,14 +1,22 @@
 import { load } from 'web-audio-beat-detector-broker';
 import { worker } from './worker/worker';
 
-const blob: Blob = new Blob([worker], { type: 'application/javascript; charset=utf-8' });
+let analyze, guess
```

Comment:
I think we need to explicitly define the types of the variables upfront. Otherwise they will be set to `any`.

```suggestion
let analyze: ReturnType<typeof load>['analyze'];
let guess: ReturnType<typeof load>['guess'];
```

But I'm feeling a little bad about it since it's basically a lie.

Reply:
Good call, I completely forgot about adding type definitions.